apps/rush/src/actions/CheckVersionsAction.ts

```diff
@@ -37,6 +38,11 @@ export default class CheckVersionsAction extends CommandLineAction {
 
     const config: RushConfiguration = RushConfiguration.loadFromDefaultLocation();
 
+    if (config.approvedPackagesPolicyEnabled) {
+      const packageReviewChecker: PackageReviewChecker = new PackageReviewChecker(config);
+      packageReviewChecker.saveCurrentDependencies();
+    }
```

Comment:
this seems weird. check-versions should have no effect

---
In reply to: [116119876](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116119876) [](ancestors = 116119876)